src/app/modules/mpi/components/paciente-relaciones.component.ts

```diff
@@ -0,0 +1,62 @@
+
+import { IPaciente } from '../../../core/mpi/interfaces/IPaciente';
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { Plex } from '@andes/plex';
+import { IPacienteRelacion } from '../interfaces/IPacienteRelacion.inteface';
+
+@Component({
+    selector: 'paciente-relaciones',
+    templateUrl: 'paciente-relaciones.html'
+})
+export class PacienteRelacionesComponent {
+    private _relaciones: IPacienteRelacion[] = [];
+    pacienteSeleccionado: IPaciente;
+
+    // Propiedades públicas
+    public listado: IPacienteRelacion[];
+    public colorRelaciones = {
+        border: '#686b6b29',
+        hover: '#00a8e099',
+        background: '##bebfbf1c'
+
+    };
+
+    /**
+     * Listado de las relaciones de un pacientes a mostrar. Acepta una lista de relaciones
+     *
+     * @type { IPacienteRelacion[]}
+     */
+    @Input()
+    get relaciones(): IPacienteRelacion[] {
+        return this._relaciones;
```

Comment:
Este get me parece que esta de mas, al input lo podes usar como this.relaciones en el componente

Reply:
es cierto, ya lo eliminé